samples/graph-activity-feed/csharp/ActivityFeedNotification/Controllers/HomeController.cs

```diff
@@ -69,12 +69,12 @@ public ActionResult TeamNotification()
 
         [HttpPost]
         [Route("SendNotificationToUser")]
-        public async Task<ActionResult> SendNotificationToUser(TaskInfo taskInfo)
+        public async Task<ActionResult> SendNotificationToUser(TaskDetails taskModuleData)
```

Comment:
can we rename `taskModuleDate` to `taskDetails` too?

Reply:
will you do this throughout the repo. Thanks

Reply:
Changed it